schedule/yast/yast_hostname/yast_hostname+dhcp_hostname.yaml

```diff
@@ -5,8 +5,10 @@ description: >
   Test validates if installation can successfully start in case of usage of
   these parameters.
 vars:
+  EXIT_AFTER_INSTALL: '1'
```

Comment:
I guess it should be `EXIT_AFTER_START_INSTALL`

Reply:
yap, it was wrong in the osd database and I didn't notice.